navigation.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
```

Comment:
UX improvement: add a border to the navbar component so that it will be visible.
![Screenshot 2022-02-20 at 1 12 03 AM](https://user-images.githubusercontent.com/79623546/154816545-3bd22fb4-89ef-4774-9382-1d9001129002.png)

Reply:
Thanks for the suggestion, will surely consider it going forward!